packages/pigment-css-react/src/utils/extendTheme.ts

```diff
@@ -71,17 +74,24 @@ export type ExtendTheme<
     vars: Options['tokens'];
     applyStyles: (
       colorScheme: Options['colorScheme'],
-      styles: CSSObject<any>,
-    ) => Record<string, CSSObject<any>>;
+      styles: CSSObject,
+    ) => Record<string, CSSObject>;
     getColorSchemeSelector: (colorScheme: Options['colorScheme']) => string;
     generateCssVars: (colorScheme?: Options['colorScheme']) => {
       css: Record<string, string | number>;
       selector: string | Record<string, any>;
     };
     unstable_sxConfig?: SxConfig;
+    unstable_sx?: (
+      props: SxProps<
+        Options['tokens'] & {
+          vars: Options['tokens'];
+        }
+      >,
+    ) => CSSObject;
   };
 
-export type Theme = ExtendTheme;
+export type Theme = Record<string, any>;
```

Comment:
I think I made a mistake on this in the previous PR. Found this issue while writing tests.